encord/objects/common.py

```diff
@@ -813,14 +796,19 @@ def _handle_wrong_number_of_found_items(
         )
 
 
+OptionAttribute = Union[RadioAttribute, ChecklistAttribute]
+OntologyElement = Union[Attribute, Option]
 AttributeTypes = Union[
     Type[RadioAttribute],
     Type[ChecklistAttribute],
     Type[TextAttribute],
     Type[Attribute],
 ]
-AttributeClasses = Union[RadioAttribute, ChecklistAttribute, TextAttribute, Attribute]
 OptionTypes = Union[Type[FlatOption], Type[NestableOption], Type[Option]]
+OntologyElementType = Union[AttributeTypes, OptionTypes]
+
+# Just kept for backwards compatibility for now
```

Comment:
It's unclear that this comment is for the next two lines and not only for the next one.